content/management/limits/index.mdx

```diff
@@ -25,6 +25,7 @@ Based on the <a href="https://www.livechat.com/pricing/" target="_blank">billing
 | Number of banned visitors     | 50      | 200   | 400      | 400        |
 | Tracked visitors              | 400     | 400   | 400      | 400        |
 | Customers history             | 2 mos.  | 1 yr. | 2 yr.    | 4 yr.      |
+| Auto Access Rules             | 0       | 200   | 200      | 200        |
```

Comment:
It's actually 200 for all plans ([source](https://livechatinc.atlassian.net/browse/API-8584))

Reply:
There is info here provided that we should set `0` for starters: https://livechatinc.atlassian.net/browse/API-8526 `Set the limit to 200 rules per license for all available plans (based on current usage in plans) - and 0 for starters` I think its worth asking Marcin to clarify this

Reply:
Mos def! Especially that I already implemented the DB changes according to the subtask description. @mdojwa which is correct?

Reply:
For future reference: it's 0 for starters after all